target-bfin/bfin-sim.c

```diff
@@ -729,7 +729,7 @@ decode_CCflag_0(DisasContext *dc, uint16_t iw0)
 
     if (opc > 4) {
         TCGv_i64 tmp64;
-        TCGCond cond;
+        TCGCond cond = 0;
```

Comment:
i've add a bunch of `return;` statements after `illegal_instruction`, so this shouldn't be needed anymore.  thanks for pointing it out.